.gitignore

```diff
@@ -1,2 +1,3 @@
 build
-out
\ No newline at end of file
+out
+.vscode
```

Comment:
tubonit: leaving a blank line at the end of the file helps with diffs (and will avoid the red warning Github gives you)